internal/messaging/response_handler.go

```diff
@@ -169,6 +120,7 @@ func (h *evmResponseHandler) handleMintResponse(ctx context.Context, response *R
 	h.logger.Debugf("Token URI JSON: %s\n", jsonPlain)
 
 	if response.MintResponse.BuyableUntil == nil || response.MintResponse.BuyableUntil.Seconds == 0 {
+		// TODO @VjeraTurk should refuse to mint?
```

Comment:
@havan should the bot respond with an error?

Reply:
If it's nil or zero, let's set it to **now+300s**. But we should also check if it's lower then **now+70s** and set it to **now+70s** if it is.